app/src/pages/admin/projectsPage/project/index.js

```diff
@@ -0,0 +1 @@
+export { ProjectInfo } from './projectInfo';
```

Comment:
Please, follow the our common naming:
`folderName` = `componentFileName`
So please rename `project` to `projectInfo`